src/ripe_rainbow/domain/logic/retail.py

```diff
@@ -55,6 +55,57 @@ def select_size(self, scale, size):
         self.interactions.click_when_possible(".size .button.button-primary.button-apply")
         self.waits.is_not_visible(".size .modal-container")
 
+    def change_customization(self, brand, model, part, material, color):
+        """
+        Makes a change to the customization of a part and checks that the pages mutates correctly, picking the right
+        active parts, materials and colors, as well as properly switching the swatches.
+
+        :param brand: The brand of the model.

```

Comment:
:type and :rtype and try to break the lines according to the typical line width

Reply:
What's our approximate line width? I was using PEP-8 max width as reference.